en/developer-docs/docs/administer/configure-developer-portal-envs.md

```diff
@@ -0,0 +1,16 @@
+# Configure Exposed Environments to Developer Portal
```

Comment:
We need these docs in PE view too?

Reply:
Yes. I'll add them also with this PR ones the content is finalized.